source/_integrations/mikrotik_bt5.markdown

```diff
@@ -0,0 +1,29 @@
+---
+title: MikroTik BT5
+description: Instructions on how to integrate MikroTik BT5 devices into Home Assistant.
+ha_category:
+  - Sensor
+ha_bluetooth: true
+ha_release: 2024.6
+ha_iot_class: Local Push
+ha_codeowners:
+  - '@anrijs'
+ha_domain: mikrotik_bt5
+ha_config_flow: true
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+---
+
+Integrates [MikroTik](https://mikrotik.com/) Bluetooth tags into Home Assistant.
+
+{% include integrations/config_flow.md %}
+
+The MikroTik BT5 integration will automatically discover devices once the [Bluetooth](/integrations/bluetooth) integration is enabled and functional.
+
```

Comment:
could you provide some information on what sensors are supported?

Reply:
It depens on https://www.home-assistant.io/integrations/bluetooth/  supported devices.
Any bluetooth adapter that can receive other BLE advertisements, like Eddystone or iBeacon, will work.

Reply:
Right, I should have been more specific. :-) What I meant was, what type of sensor entities does this integration provide.

Reply:
I have added what sensors each model supports